documentation/modules/ROOT/pages/characteristics.adoc

```diff
@@ -332,6 +332,19 @@ in the `elements` list are used for construction. When the Property using the St
 Characteristic also defines an `exampleValue`, construction and deconstruction of the value are
 tested when validating the model.
 
+[[reference-characteristic]]
+=== Reference
```

Comment:
References is not included in the Characteristics Instances overview diagram. Please also adjust characteristics-instances.dot then regenerate the .svg (see [instructions](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model?tab=readme-ov-file#build-diagrams)).